README.md

```diff
@@ -203,6 +203,9 @@
 * [**Python**: *Building a basic HTTP Server from scratch in Python*](http://joaoventura.net/blog/2017/python-webserver/)
 * [**Ruby**: *Building a simple websockets server from scratch in Ruby*](http://blog.honeybadger.io/building-a-simple-websockets-server-from-scratch-in-ruby/)
 
+#### Build your own `CI Server`
+* [**Ruby**: *Writing a CI Server*](https://developer.github.com/v3/guides/building-a-ci-server/)
```

Comment:
Thank you for your submission. Can you please move this to "Uncategorized" for now? As soon as there are 3 or more tutorials about the same topic, I'll move it into its own category.

Reply:
Sure, moved the proposal the ruby items in `Uncategorized`